lib/msal-node/src/encode/Base64Encode.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export class Base64Encode {
```

Comment:
Why have two seperate classes ? why not just have a Base64Utils or EncodingUtils

Reply:
+1 for `EncodingUtils`

Reply:
I wondered the same, I was constrained by the interface provided by `msal-common` and the pattern in `msal-browser`, this would need to change in all three components if we want it this way.

Reply:
Done. We have a class:`/utils/EncodeUtils.ts` which does the Base64 encoding and decoding.